packages/api/src/main.ts

```diff
@@ -1,10 +1,12 @@
 import 'dotenv/config';
 import { NestFactory } from '@nestjs/core';
+import cookieParser from 'cookie-parser';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  app.enableCors();
+  app.enableCors({ origin: 'http://localhost:8080', credentials: true });
+  app.use(cookieParser('secret'));
```

Comment:
As we adding those steps we can also add [helmet](https://docs.nestjs.com/security/helmet) and [rate limit](https://docs.nestjs.com/security/rate-limiting), but for sure it can be added in separate PR.